lib/runner.ts

```diff
@@ -1,5 +1,6 @@
 import {EventEmitter} from 'events';
 import * as q from 'q';
+import {Session} from 'selenium-webdriver';
 import * as util from 'util';
 
 import {ProtractorBrowser} from './browser';
```

Comment:
nit: below on line 17, prefer `const webdriver = ...`.  Or, if you can address my other nit from this file, remove the declaration entirely.